README.md

```diff
@@ -55,12 +55,37 @@ dbt debug
 
 ### Load source data
 
-To copy source data files from Digital Ocean:
+To copy source data files from Digital Ocean, we can use `curl` for public files:
 
 ```bash
 # for public files
 curl --remote-name https://BUCKET.URL.com/folder/folder/data.csv
+```
+
+For non-public files like our CSVs in `/edm/distribution/`, we can use [minio](https://github.com/minio/minio).
+
+#### Using brew:
+
+Install
+```bash
+brew install minio/stable/mc
+```
+
+Add DO Spaces to the `mc` configuration
+```bash
+mc alias set spaces $DO_SPACES_ENDPOINT $DO_SPACES_ACCESS_KEY_ID $DO_SPACES_SECRET_ACCESS_KEY
+```
+We use `spaces` here but you can name the alias anything. When you run `mc config host list` you should see the newly added host with credentials from your `.env`.
+
+Copy CSV files
+```bash
+mc cp spaces/${DO_SPACES_BUCKET_DISTRIBUTIONS}/dcp_pluto/23v3/pluto.csv pluto.csv
+mc cp spaces/${DO_SPACES_BUCKET_DISTRIBUTIONS}/dcp_pluto/23v3/attachments/zoning_districts.csv zoning_districts.csv
+mc cp spaces/${DO_SPACES_BUCKET_DISTRIBUTIONS}/dcp_pluto/23v3/attachments/source_data_versions.csv source_data_versions.csv
+```
 
+#### Using curl directly
+```bash
```

Comment:
if you wanna delete the lines related to minio below, go for it

Reply:
Okay, I'll wait for Bryan / David to try this out first. I'm realizing that we might actually wanna keep `mc config host add` because I'm able to get it to work now ??

<img width="1287" alt="image" src="https://github.com/NYCPlanning/ae-data-flow/assets/7049943/22b228e5-a55f-4a61-96fb-4e64f9d408dc">

Reply:
I think the canonical way is to use `mc config host add` but if alias works rn, I'll delete and we can bring back if needed